starknet_py/net/account/account_client.py

```diff
@@ -321,7 +329,7 @@ async def sign_transaction(
         calls: Calls,
         max_fee: Optional[int] = None,
         auto_estimate: bool = False,
-        version: Optional[int] = None,
+        version: int = 0,
```

Comment:
We should default to version of the account

Reply:
It's not possible to default to `self` argument in function declaration but I added some default logic to the method.